arch/xtensa/src/esp32/esp32_rtc_gpio.c

```diff
@@ -127,8 +128,9 @@ static inline bool is_valid_rtc_gpio(uint32_t rtcio_num)
  ****************************************************************************/
 
 #ifdef CONFIG_ESP32_RTCIO_IRQ
-static void rtcio_dispatch(int irq, uint32_t status, uint32_t *context)
+static void rtcio_dispatch(int irq, uint32_t *reg_status)
```

Comment:
"Input Parameters" section in function documentation needs to be updated.

Reply:
Oh yes, I forgot about it. Thanks!